DocumentDataAPI/DocumentDataAPITests/Data/Repositories/NpgDocumentContentRepositoryIntegrationTests.cs

```diff
@@ -44,7 +44,8 @@ public void Update_UpdatesRow()
         string? actual = repository.Get(1).Content;
 
         // Assert
-        actual.Should().Be(expected, "because the content of the document with id 1 was updated to (1, 'Test Content')");
+        actual.Should().Be(expected,
```

Comment:
```suggestion
        actual.Should().BeEquivalentTo(expected,
```
Ved godt at det ikke er jer der har lavet den her, men det er mere læsbart med den method jeg har foreslået! De gør det samme, Be ligner bare en sammenligning på reference hvor det i virkeligheden bare er ækvivalens der bliver tested